main.go

```diff
@@ -223,6 +175,15 @@ func main() { //nolint:funlen
 		os.Exit(1)
 	}
 
+	// Check if user opted for disabling DSC configuration
+	_, disableDSCConfig := os.LookupEnv("DISABLE_DSC_CONFIG")
+	if !disableDSCConfig {
+		if err = upgrade.CreateDefaultDSCI(setupClient, platform, dscApplicationsNamespace, dscMonitoringNamespace); err != nil {
```

Comment:
to call it as part of `upgrade` package is a bit confusion but i understand it is calling the `CreateDefaultDSCI` function

Reply:
Yeah I kept the function in same package as `createDefaultDSC`